components/places/src/places.udl

```diff
@@ -40,7 +65,7 @@ message HistoryVisitInfosWithBound {
  * Note that these docs comments are internal, and don't necessarily impact the actual
  * API we expose to Kotlin/Swift (this is particularly true around reads).
  */
-message BookmarkNode {
+dictionary BookmarkNode {
     /**
```

Comment:
FWIW, autofill has landed on using 2 versions of the struct - one for "incoming" (eg, "create an item" where guid might be null) and one for outgoing (eg "fetch an item" where guid is never null). I assume this is used in both directions which is why everything is nullable.

(Not sure why I'm noting this TBH :)

Reply:
Right, yes, I think separate "incoming" and "outgoing" would make a big difference here.